apps/collector-proxy/src/workers/post-schema.ts

```diff
@@ -18,7 +18,7 @@ export async function postSchemaWorker(data: PostSchema["body"]) {
 
     const hash = hashSchema(executableSchema);
 
-    const foundSchema = await prisma.schema.findFirst({
+    const foundSchema = await client.schema.findFirst({
```

Comment:
@danstarns are we making the assumption here that the GQL Debugger client will have the same API as Prisma?

Reply:
No, the api will mold itself as we add more connectors, but seeming that SQLite(prisma) is the only connector at this time its making it easier to port the code to the client.